### ISSUE TITLE
grid-template-columns is not working

### ISSUE THREAD

**Issue by HeBuTy** [richthegeek/phpsass#193]
I want to use grids but this style is not working correctly `grid-template-columns: repeat(auto-fill,minmax(260px,1fr));`

**Comment by richthegeek** [comment]
@HeBuTy hi, this project is not really maintained any more. `grid-template-columns` in particular are something that came into the CSS spec after I stopped working on it.

The problem will be that the repeat fn is not whitelisted so it's trying to execute it.

It should be resolvable by using string interpolation, e.g.

`grid-template-columns: #{repeat(auto-fill, minmax(260px, 1fr))};`

Alternatively if possible I'd recommend using libsass or some other maintained (although non-PHP) compiler.

**Comment by joejoseph00** [comment]
I still have a client using phpsass, it works well, however also compass which is easy to install in Ubuntu works well also.
phpsass is handy though for simplifying deployments, has the advantage that everything can be done in php.

**Comment by joejoseph00** [comment]
but ya for grid-template-columns, feel free to fork phpsass and work on this and make a pull request, perhaps @richthegeek will add some more project maintainers (like me?)

**Comment by joejoseph00** [comment]
check the forks of phpsass, perhaps someone else has already done this.
https://github.com/richthegeek/phpsass/network/members

**Comment by richthegeek** [comment]
@joejoseph00 happy to add you as a maintainer as otherwise this is basically abandonware 🙃 

Agree that maybe a fork has taken this further. This was originally a fork of `phamlp` after all!

**Comment by joejoseph00** [comment]
ok thanks, I'll review others contributions and babysit this as I've already been doing, however, ya it just works actually quite well so other than a few minor glitches, it's been working for years for one of my clients and phpsass is good for certain types of clients that cannot be bothered with installing stuff, also there's  a Drupal module prepro and sassy that make using this easy.

**Comment by richthegeek** [comment]
@joejoseph00 yeah i wrote those modules... again, abandoned after I moved job 6 or so years ago. Pretty crazy to hear this is still being used!

**Comment by joejoseph00** [comment]
Ya , there's still almost 2000 installs of those modules.

**Comment by joejoseph00** [comment]
@richthegeek  it still says I don't have write access, there's a merge request for php 7.2 / php 7.3 compatibility
This branch has no conflicts with the base branch
**Only those with write access to this repository can merge pull requests.**

**Comment by joejoseph00** [comment]
https://github.com/richthegeek/phpsass/pull/192

**Comment by joejoseph00** [comment]
ah, helps if i "ACCEPT" the invitation, ok it should be good now

**Comment by joejoseph00** [comment]
Hmm, looks like I lack access rights to enable the travis .ci 
I enabled travis in my fork and am waiting for the test results

https://travis-ci.org/github/joejoseph00/phpsass/builds/693524613
